welcome.tex

```diff
@@ -1,6 +1,6 @@
 \newpage
 \enlargethispage{1\baselineskip}
-\section*{Welcome to Heidelberg and State of the Map 2019} \label{welcome}
+\section*{Welcome to Heidelberg and State of the Map \numeral{2019}} \label{welcome}
```

Comment:
Please let it be as it is.

Reply:
see my answer [above](https://github.com/osmfoundation/sotm2019-booklet/pull/1#discussion_r313829333)